### PR TITLE
I2P support for search, Imgur and Wikipedia

### PR DESCRIPTION
**Pull request by BobIsMyManager**

### CODE REVIEW

**Comment by BobIsMyManager** [comment]
Is there anything that needs to be done?

**Review by BobIsMyManager — COMMENTED** [review]
All good!

File: src/assets/javascripts/helpers/imgur.js [hunk 1]
```diff
@@ -59,6 +79,14 @@ function setRimgoTorRedirectsChecks(val) {
     console.log("rimgoTorRedirectsChecks: ", val)
 }
 
+let rimgoI2PRedirectsChecks;
+const getRimgoI2PRedirectsChecks = () => rimgoI2PRedirectsChecks;
+function setRimgoI2PRedirectsChecks(val) {
+    rimgoI2PRedirectsChecks = val;
```
[the remark this answers is not in the document]
Comment (BobIsMyManager) [reply src/assets/javascripts/helpers/imgur.js:85]: Oh its just that in the matrix room you told me to use I2P instead og I2p

**Comment by ManeraKai** [comment]
Looks good to me. Thanks!

**Pull request merged**